### PR TITLE
Bxmsdoc 8029 Certifications

### PR DESCRIPTION
**Pull request by briandooley**
This PR covers

[BXMSDOC-8027](https://issues.redhat.com/browse/BXMSDOC-8027)
[BXMSDOC-8028](https://issues.redhat.com/browse/BXMSDOC-8028)
[BXMSDOC-8029](https://issues.redhat.com/browse/BXMSDOC-8029)
[BXMSDOC-8058](https://issues.redhat.com/browse/BXMSDOC-8058)
[BXMSDOC-8061](https://issues.redhat.com/browse/BXMSDOC-8061)

### CODE REVIEW

File: doc-content/enterprise-only/installation/clustering-bc-install-proc.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@ After you have configured a {EAP} cluster and repackaged {CENTRAL}, you must sta
 * The repackaged `{URL_COMPONENT_CENTRAL}.war` file is available.
 
 .Procedure
-. Add {EAP} management users to the master node of the cluster (where you configured the `domain.xml` file) as described in the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/{EAP_VERSION}/html-single/configuration_guide/[_Red Hat JBoss EAP 7.3 Configuration Guide_].
+. Add {EAP} management users to the master node of the cluster (where you configured the `domain.xml` file) as described in the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/{EAP_VERSION}/html-single/configuration_guide/[_Red Hat JBoss EAP 7.4 Configuration Guide_].
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-bc-install-proc.adoc]: I searched for this doc and it doesnt appear to be used in any assembly. Double check by searching for clustering-bc-install-proc.adoc  in the repo. If you don't find it in an include statement, then delete.

File: doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 For more efficient searching across cluster nodes, install {DATAGRID} and configure it for the {PRODUCT} clustered environment.
 
-For information about {DATAGRID} advanced installation and configuration options and {DATAGRID} modules for {EAP}, see the https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html-single/red_hat_data_grid_user_guide/index#[_{DATAGRID} User Guide_].
+For information about {DATAGRID} advanced installation and configuration options and {DATAGRID} modules for {EAP}, see the https://access.redhat.com/documentation/en-us/red_hat_data_grid/{DATAGRID_VERSION}/html-single/red_hat_data_grid_user_guide/index#[_{DATAGRID} User Guide_].
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc]: Note that this URL does not work. I checked the DG 8.0 doc library (https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.0) and there isn't a User Guide anymore. Find out what doc you should reference now.

Comment (briandooley) [reply doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc]: Doc to reference is https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.1/html/data_grid_server_guide/index .

File: doc-content/enterprise-only/installation/clustering-bc-install-proc.adoc (hunk 1 above)
Comment (briandooley) [reply doc-content/enterprise-only/installation/clustering-bc-install-proc.adoc]: Deleted.

**Review by Rikkola — APPROVED** [review]

File: _artifacts/document-attributes.adoc [hunk 3]
```diff
@@ -97,20 +97,20 @@ endif::OP[]
 :CODEREADY_STUDIO_VERSION: 12.17
 :CODEREADY_STUDIO_VERSION_LONG: 12.17.0
 :DATAGRID: Red Hat Data Grid
-:DATAGRID_VERSION: 7.3
-:DATAGRID_VERSION_LONG: 7.3.6
+:DATAGRID_VERSION: 8.0
```
Comment (tomasdavidorg) [inline _artifacts/document-attributes.adoc]:
```suggestion
:DATAGRID_VERSION: 8.1
```

File: _artifacts/document-attributes.adoc [hunk 4]
```diff
@@ -97,20 +97,20 @@ endif::OP[]
...
+:DATAGRID_VERSION_LONG: 8.0.0
```
Comment (tomasdavidorg) [inline _artifacts/document-attributes.adoc]:
```suggestion
:DATAGRID_VERSION_LONG: 8.1.0
```

File: doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc [hunk 5]
```diff
@@ -14,10 +14,10 @@ NOTE: Do not install {DATAGRID} on the same node as {CENTRAL}.
 . Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
 +
 * *Product:* Data Grid
-* *Version:* 7.3
+* *Version:* {DATAGRID_VERSION}
 
-. Download and extract the *Red Hat JBoss Data Grid 7.3.0 Server* (`jboss-datagrid-7.3.0-1-server.zip`) installation file to the preferred location on your system. In the following examples, the extracted directory is referred to as `__JDG_HOME__`.
-. Update {DATAGRID} to the latest version. For more information, see {DATAGRID} https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html-single/red_hat_data_grid_user_guide/index#[_{DATAGRID} User Guide_].
+. Download and extract the *Red Hat JBoss Data Grid {DATAGRID_VERSION_LONG} Server* (`jboss-datagrid-{DATAGRID_VERSION}.0-1-server.zip`) installation file to the preferred location on your system. In the following examples, the extracted directory is referred to as `__JDG_HOME__`.
```
Comment (tomasdavidorg) [inline doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc]:
```suggestion
. Download and extract the *Red Hat Data Grid {DATAGRID_VERSION_LONG} Server* (`redhat-datagrid-{DATAGRID_VERSION_LONG}-server.zip`) installation file to the preferred location on your system. In the following examples, the extracted directory is referred to as `__JDG_HOME__`.
```

File: _artifacts/document-attributes.adoc [hunk 6]
```diff
@@ -97,20 +97,20 @@ endif::OP[]
...
 :FUSE: Fuse
 :FUSE_LONG: Red Hat Fuse
 :FUSE_VERSION: 7.9
 :FUSE_EAP_VERSION: 7.4
 :SPRING_BOOT_VERSION: 2.3.4
 :JWS: Red Hat JBoss Web Server
-:JWS_VERSION: 5.4
-:JWS_VERSION_LONG: 5.4.0
+:JWS_VERSION: 5.5
+:JWS_VERSION_LONG: 5.5.0
```
Comment (tomasdavidorg) [inline _artifacts/document-attributes.adoc]: @sutaakar Are these versions correct?

**Review by tomasdavidorg — CHANGES_REQUESTED** [review]

File: _artifacts/document-attributes.adoc (hunk 6 above)
Comment (sutaakar) [reply _artifacts/document-attributes.adoc]: there is a 5.5.1 version already available, we may upgrade to it.

Comment (briandooley) [reply _artifacts/document-attributes.adoc]: > there is a 5.5.1 version already available, we may upgrade to it.

I'm afraid I'm going to need a something a bit more concrete, guys. ;o) Should I keep it at 5.5.0 or change to 5.5.1?

Comment (sutaakar) [reply _artifacts/document-attributes.adoc]: @briandooley Please change it to 5.5.1.

**Comment by briandooley** [comment]
@tomasdavidorg I have applied the changes that have been requested. Can you take a take a final (hopefully) look and approve? Here are some links for the rendered versions of the [RHPAM](http://file.ork.redhat.com/~bdooley/BXMSDOC-8029-RHPAM/) and [RHDM](http://file.ork.redhat.com/~bdooley/BXMSDOC-8029-RHDM/).

**Review by tomasdavidorg — APPROVED** [review]

**Pull request merged**